e2e `test_070_run_sdntrace_untagged_vlan` is failing  (April 10 nightly exec)

@gretelliz e2e test `test_070_run_sdntrace_untagged_vlan` is still failing on CI, can you fix this? 

```
tests/test_e2e_40_sdntrace.py::TestE2ESDNTrace::test_070_run_sdntrace_untagged_vlan: 2023-04-10,06:45:44.852582 - 2023-04-10,06:45:46.065119
cls = <tests.test_e2e_40_sdntrace.TestE2ESDNTrace object at 0x7f1bad746fa0>
    def test_070_run_sdntrace_untagged_vlan(cls):
        """Run SDNTrace to test /traces endpoint when vlan is untagged in evc"""
    
        cls.create_evc("untagged", interface_a="00:00:00:00:00:00:00:02:1", interface_z="00:00:00:00:00:00:00:03:1")
        payload = [
                    {
                        "trace": {
                            "switch": {
                                "dpid": "00:00:00:00:00:00:00:02",
                                "in_port": 1
                            }
                        }
                    }
        ]
    
        api_url = KYTOS_API + '/amlight/sdntrace_cp/traces'
        response = requests.put(api_url, json=payload)
        assert response.status_code == 200, response.text
        data = response.json()
        list_results = data["result"]
    
>       assert len(list_results[0]) == 2
E       AssertionError: assert 1 == 2
E        +  where 1 = len([{'dpid': '00:00:00:00:00:00:00:02', 'out': None, 'port': 1, 'time': '2023-04-10 06:45:45.992499', ...}])
tests/test_e2e_40_sdntrace.py:797: AssertionError
rerun: 1
tests/test_e2e_40_sdntrace.py::TestE2ESDNTrace::test_070_run_sdntrace_untagged_vlan: 2023-04-10,06:45:46.241127 - 2023-04-10,06:45:46.424571
cls = <tests.test_e2e_40_sdntrace.TestE2ESDNTrace object at 0x7f1bad746fa0>
    def test_070_run_sdntrace_untagged_vlan(cls):
        """Run SDNTrace to test /traces endpoint when vlan is untagged in evc"""
    
>       cls.create_evc("untagged", interface_a="00:00:00:00:00:00:00:02:1", interface_z="00:00:00:00:00:00:00:03:1")
tests/test_e2e_40_sdntrace.py:779: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
vlan_id = 'untagged', interface_a = '00:00:00:00:00:00:00:02:1'
interface_z = '00:00:00:00:00:00:00:03:1'
    @staticmethod
    def create_evc(vlan_id, interface_a="00:00:00:00:00:00:00:01:1", interface_z="00:00:00:00:00:00:00:0a:1"):
        payload = {
            "name": "Vlan_%s" % vlan_id,
            "enabled": True,
            "dynamic_backup_path": True,
            "uni_a": {
                "interface_id": interface_a,
                "tag": {"tag_type": 1, "value": vlan_id}
            },
            "uni_z": {
                "interface_id": interface_z,
                "tag": {"tag_type": 1, "value": vlan_id}
            }
        }
        api_url = KYTOS_API + '/kytos/mef_eline/v2/evc/'
        response = requests.post(api_url, json=payload)
>       assert response.status_code == 201, response.text
E       AssertionError: {"code":409,"description":"The EVC already exists.","name":"Conflict"}
E         
E       assert 409 == 201
E        +  where 409 = <Response [409]>.status_code
tests/test_e2e_40_sdntrace.py:46: AssertionError
tests/test_e2e_40_sdntrace.py::TestE2ESDNTrace::test_070_run_sdntrace_untagged_vlan: 2023-04-10,06:45:46.515972 - 2023-04-10,06:45:46.569[379](https://gitlab.ampath.net/kytos/kytos-end-to-end-tester/-/jobs/42821#L379)
=========================== rerun test summary info ============================
RERUN tests/test_e2e_40_sdntrace.py::TestE2ESDNTrace::test_070_run_sdntrace_untagged_vlan
RERUN tests/test_e2e_40_sdntrace.py::TestE2ESDNTrace::test_070_run_sdntrace_untagged_vlan
=========================== short test summary info ============================
FAILED tests/test_e2e_40_sdntrace.py::TestE2ESDNTrace::test_070_run_sdntrace_untagged_vlan - AssertionError: {"code":[409](https://gitlab.ampath.net/kytos/kytos-end-to-end-tester/-/jobs/42821#L409),"description":"The EVC already exists.","name":"Conflict"}
  
assert 409 == 201
 +  where 409 = <Response [409]>.status_code
= 1 failed, 202 passed, 2 skipped, 17 xfailed, 5 xpassed, 799 warnings, 2 rerun in 11492.58s (3:11:32) =
```

--- a/tests/test_e2e_40_sdntrace.py
+++ b/tests/test_e2e_40_sdntrace.py
@@ -776,22 +776,21 @@ class TestE2ESDNTrace:
     def test_070_run_sdntrace_untagged_vlan(cls):
         """Run SDNTrace to test /traces endpoint when vlan is untagged in evc"""
 
-        payload = {
-            "name": "Vlan_untagged",
-            "enabled": True,
-            "dynamic_backup_path": True,
-            "uni_a": {
-                "interface_id": "00:00:00:00:00:00:00:02:1",
-                "tag": {"tag_type": 1, "value": "untagged"}
-            },
-            "uni_z": {
-                "interface_id": "00:00:00:00:00:00:00:03:1",
-                "tag": {"tag_type": 1, "value": "untagged"}
-            }
-        }
-        api_url = KYTOS_API + '/kytos/mef_eline/v2/evc/'
-        response = requests.post(api_url, json=payload)
-        assert response.status_code in [201, 409], response.text
+        api_url = KYTOS_API + '/kytos/mef_eline/v2/evc/'  
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        data = response.json()
+        
+        uni_a = {'interface_id': '00:00:00:00:00:00:00:02:1', 'tag': {'tag_type': 1, 'value': "untagged"}}
+        uni_z = {'interface_id': '00:00:00:00:00:00:00:03:1', 'tag': {'tag_type': 1, 'value': "untagged"}}
+        for circuit_id, circuit in data.items():
+            if uni_a in (circuit['uni_a'], circuit['uni_z']) or uni_z in (circuit['uni_a'], circuit['uni_z']):
+                api_url = KYTOS_API + f'/kytos/mef_eline/v2/evc/{circuit_id}' 
+                response = requests.delete(api_url)
+                assert response.status_code == 200, response.text
+
+        cls.create_evc("untagged", interface_a="00:00:00:00:00:00:00:02:1", interface_z="00:00:00:00:00:00:00:03:1")        
+        time.sleep(10)
 
         payload = [
                     {
@@ -817,23 +816,8 @@ class TestE2ESDNTrace:
 
     def test_075_run_sdntrace_any_vlan(cls):
         """Run SDNTrace to test /traces endpoint when vlan is any in evc"""
-       
-        payload = {
-            "name": "Vlan_any",
-            "enabled": True,
-            "dynamic_backup_path": True,
-            "uni_a": {
-                "interface_id": "00:00:00:00:00:00:00:02:1",
-                "tag": {"tag_type": 1, "value": "any"}
-            },
-            "uni_z": {
-                "interface_id": "00:00:00:00:00:00:00:03:1",
-                "tag": {"tag_type": 1, "value": "any"}
-            }
-        }
-        api_url = KYTOS_API + '/kytos/mef_eline/v2/evc/'
-        response = requests.post(api_url, json=payload)
-        assert response.status_code in [201, 409], response.text
+
+        cls.create_evc("any", interface_a="00:00:00:00:00:00:00:02:1", interface_z="00:00:00:00:00:00:00:03:1")
         time.sleep(10)
 
         payload = [

--- a/tests/test_e2e_40_sdntrace.py
+++ b/tests/test_e2e_40_sdntrace.py
@@ -776,7 +776,23 @@ class TestE2ESDNTrace:
     def test_070_run_sdntrace_untagged_vlan(cls):
         """Run SDNTrace to test /traces endpoint when vlan is untagged in evc"""
 
-        cls.create_evc("untagged", interface_a="00:00:00:00:00:00:00:02:1", interface_z="00:00:00:00:00:00:00:03:1")
+        payload = {
+            "name": "Vlan_untagged",
+            "enabled": True,
+            "dynamic_backup_path": True,
+            "uni_a": {
+                "interface_id": "00:00:00:00:00:00:00:02:1",
+                "tag": {"tag_type": 1, "value": "untagged"}
+            },
+            "uni_z": {
+                "interface_id": "00:00:00:00:00:00:00:03:1",
+                "tag": {"tag_type": 1, "value": "untagged"}
+            }
+        }
+        api_url = KYTOS_API + '/kytos/mef_eline/v2/evc/'
+        response = requests.post(api_url, json=payload)
+        assert response.status_code in [201, 409], response.text
+
         payload = [
                     {
                         "trace": {
@@ -801,7 +817,23 @@ class TestE2ESDNTrace:
 
     def test_075_run_sdntrace_any_vlan(cls):
         """Run SDNTrace to test /traces endpoint when vlan is any in evc"""
-        cls.create_evc("any", interface_a="00:00:00:00:00:00:00:02:1", interface_z="00:00:00:00:00:00:00:03:1")
+       
+        payload = {
+            "name": "Vlan_any",
+            "enabled": True,
+            "dynamic_backup_path": True,
+            "uni_a": {
+                "interface_id": "00:00:00:00:00:00:00:02:1",
+                "tag": {"tag_type": 1, "value": "any"}
+            },
+            "uni_z": {
+                "interface_id": "00:00:00:00:00:00:00:03:1",
+                "tag": {"tag_type": 1, "value": "any"}
+            }
+        }
+        api_url = KYTOS_API + '/kytos/mef_eline/v2/evc/'
+        response = requests.post(api_url, json=payload)
+        assert response.status_code in [201, 409], response.text
         time.sleep(10)
 
         payload = [

